packages/providers/src/sdk/utils.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Stores promises associated with message nonces.
+ */
+export const promises: {
```

Comment:
Let's keep this inside sdk without exporting. This is internal data and it shouldn't be exported.
As a dev I can just `import { promises } from "@cryptkeeperzk/providers/dist/src/sdk/utils"` from outside the repo.

Reply:
Good point. :ok_hand: